index.html

```diff
@@ -11,35 +11,40 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button type="button" id="open-menu-mobile" onclick="openMenuMobile()">
         <i class="fas fa-bars"></i>
       </button>
+
+      <nav id="mobile-nav">
+        <button type="button" id="close-menu-mobile">
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="mobile-nav-link-list">
+          <li><a class="mobile-nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="mobile-nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="mobile-nav-link" href="#contact">
+              Contact <i class="far fa-envelope"></i>
+            </a>
+          </li>
```

Comment:
- [ ] According to the project requirements, when the user clicks any of the menu options the menu should disappear as well. please implement it similar to how I described in the previous comment regarding the event handler.